product_operating_unit/__manifest__.py

```diff
@@ -18,6 +18,7 @@
     "data": [
         "security/product_template_security.xml",
         "views/product_template_view.xml",
+        "data/product_operating_unit_data.xml",
```

Comment:
I don't find it in the guidelines, but I think the order tends to be data, views, security

Reply:
OK ─ I think the order you mention is the usual one, yes.